src/main/scripts/cdp-listend

```diff
@@ -239,6 +247,7 @@ cdp-listend [OPTIONS]
   --fetch_smear=SECONDS   fetch run smearing time
   --nch_smear=SECONDS     nch run smearing time\n";
   --facility              syslog facility
+  --hostname=fqdn         hostname/IP to listen on (use 0.0.0.0 to listen on all IPs)
```

Comment:
Listening on 0.0.0.0 is potentially a security risk. I think it was a feature of the old code that it was so restrictive so it was not possible to accidentally start listening on external interfaces. I don't like encoding configuration policy in code (that's what PAN is for!) but I feel reticent about this addition to the help text.